buildSrc/build.gradle

```diff
@@ -14,4 +14,5 @@ dependencies {
     implementation libs.tomlj
     implementation libs.japicmp.gradle.plugin
     implementation libs.native.gradle.plugin
+    implementation("org.sonatype.gradle.plugins:scan-gradle-plugin:3.0.0")
```

Comment:
Why are there no libs versions in the directory?

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/11465/commits/160042426b3e0adc5ceec699bf5b0c36667a7025